topydo/lib/PrettyPrinterFilter.py

```diff
@@ -84,15 +86,40 @@ def filter(self, p_todo_str, p_todo):
 
         return p_todo_str
 
+
 class PrettyPrinterIndentFilter(PrettyPrinterFilter):
+
     """ Adds indentation to the todo item. """
-    def __init__(self, p_indent=0):
+
+    def __init__(self, p_indent=0, p_max_lines=None):
         super(PrettyPrinterIndentFilter, self).__init__()
         self.indent = p_indent
+        self.max_lines = p_max_lines
 
     def filter(self, p_todo_str, _):
         """ Applies the indentation. """
-        return ' ' * self.indent + p_todo_str
+        try:
+            return(textwrap.fill(p_todo_str,
+                                 initial_indent=' '*self.indent,
+                                 subsequent_indent=' '*(10 + self.indent),
```

Comment:
The 10 seems to be a magic number to me, which needs a clarification in a comment.